src/google-maps/README.md

```diff
@@ -0,0 +1,127 @@
+# Angular Google Maps component
+
+This component provides a Google Maps Angular component that implements the
+[Google Maps JavaScript API](https://developers.google.com/maps/documentation/javascript/tutorial).
+File any bugs against the [angular/components repo](https://github.com/angular/components/issues).
+
+## Installation
+
+To install, run `npm install @angular/google-maps`.
+
+## Loading the API
+
+- First follow [these steps](https://developers.google.com/maps/gmp-get-started) to get an API key that can be used to load Google Maps.
+- Make sure that the API is loaded before bootstrapping the `GoogleMap` component. See the [example](#example) below.
```

Comment:
We should point to the main docs for loading the web API here:
https://developers.google.com/maps/documentation/javascript/tutorial#Loading_the_Maps_API

Reply:
Done.